src/surge-xt/gui/overlays/LuaEditors.cpp

```diff
@@ -1010,7 +1021,455 @@ struct WavetablePreviewComponent : juce::Component
     Surge::GUI::Skin::ptr_t skin;
 };
 
-WavetableEquationEditor::WavetableEquationEditor(SurgeGUIEditor *ed, SurgeStorage *s,
+struct WavetableScriptControlArea : public juce::Component,
+                                    public Surge::GUI::SkinConsumingComponent,
+                                    public Surge::GUI::IComponentTagValue::Listener,
+                                    public juce::ComboBox::Listener,
+                                    public juce::TextEditor::Listener
+{
+    enum tags
+    {
+        tag_select_tab = 0x597500,
```

Comment:
We probably want to use a different value here than other control areas just in case. Like just choose some other random starting number. In case messages do cross (which they should not)

Reply:
ha sorry i should have checked.

Reply:
No worries, I'm just glad I did it right :)